### PR TITLE
Updates nbs markup

### PR DESCRIPTION
**Pull request by nc24**
*Description of changes:* updated  markup 2 notebooks for labs 2 and 3. In lab 2, added message to ignore warnings when running import/ and in lab 3 added reference to connect to EMR cluster similar to lab2

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: workshop/02_SentimentAnalysis/00_sparkmagic-sentiment-analysis.ipynb [hunk 1]
```diff
@@ -91,7 +91,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Next, we will import libraries and set global definitions"
+    "Next, we will import libraries and set global definitions\n",
+    "\n",
+    "Please ignore warnings about **PkgResourceDeprecation** and **invalid version** and **sagemaker.deprecations** when you run the cell below\n"
```
Comment (seanpmorgan) [inline workshop/02_SentimentAnalysis/00_sparkmagic-sentiment-analysis.ipynb]: This appears to be fixed in the most recent versions of the kernel. Can you confirm?
image

File: workshop/03_TimeSeries/00_forecasting_with_deepAR.ipynb [hunk 2]
```diff
@@ -695,19 +705,21 @@
  "metadata": {
   "instance_type": "ml.t3.medium",
   "kernelspec": {
-   "display_name": "PySpark (SparkMagic)",
+   "display_name": "Python 3 (ipykernel)",
```
Comment (seanpmorgan) [inline workshop/03_TimeSeries/00_forecasting_with_deepAR.ipynb]:
```suggestion
   "display_name": "PySpark (SparkMagic)",
```

File: workshop/03_TimeSeries/00_forecasting_with_deepAR.ipynb [hunk 3]
```diff
@@ -695,19 +705,21 @@
...
    "language": "python",
-   "name": "pysparkkernel__SAGEMAKER_INTERNAL__arn:aws:sagemaker:us-west-2:236514542706:image/sagemaker-sparkmagic"
+   "name": "python3"
```
Comment (seanpmorgan) [inline workshop/03_TimeSeries/00_forecasting_with_deepAR.ipynb]:
```suggestion
   "name": "pysparkkernel__SAGEMAKER_INTERNAL__arn:aws:sagemaker:us-west-2:236514542706:image/sagemaker-sparkmagic"
```

File: workshop/03_TimeSeries/00_forecasting_with_deepAR.ipynb [hunk 4]
```diff
@@ -695,19 +705,21 @@
...
   },
   "language_info": {
    "codemirror_mode": {
-    "name": "python",
+    "name": "ipython",
```
Comment (seanpmorgan) [inline workshop/03_TimeSeries/00_forecasting_with_deepAR.ipynb]:
```suggestion
    "name": "python",
```

File: workshop/03_TimeSeries/00_forecasting_with_deepAR.ipynb [hunk 5]
```diff
@@ -695,19 +705,21 @@
...
     "version": 3
    },
    "file_extension": ".py",
    "mimetype": "text/x-python",
-   "name": "pyspark",
-   "pygments_lexer": "python3"
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.9.12"
```
Comment (seanpmorgan) [inline workshop/03_TimeSeries/00_forecasting_with_deepAR.ipynb]:
```suggestion
   "name": "pyspark",
   "pygments_lexer": "python3"
```

**Review by seanpmorgan — CHANGES_REQUESTED** [review]

**Comment by nc24** [comment]
approved

**Pull request closed without merging**